frontend/src/service/config.service.ts

```diff
@@ -0,0 +1,4 @@
+import axios from 'axios'
+
+const URL = 'http://localhost:3000/verifyBoard'
```

Comment:
The URL is a good pattern? PLZ learn Restful API best practice

Reply:
delete json-server and replace with a fake road address https://jsonplaceholder.typicode.com/posts to impersonation request